### PR TITLE
Update bundle

### PR DESCRIPTION
**Pull request by praveenkumar**
- cut 1.37.0
- update to 4.9.10
- Fix CI changes

### CODE REVIEW

**Comment by praveenkumar** [comment]
/hold 

Till the CI green.

File: images/openshift-ci/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ WORKDIR /go/src/github.com/code-ready/crc
 COPY . .
 RUN make cross
```
Comment (gbraad) [inline images/openshift-ci/Dockerfile:7]: So what does builder do?

File: images/openshift-ci/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ WORKDIR /go/src/github.com/code-ready/crc
...
 
-FROM registry.centos.org/centos/centos:7
+FROM registry.centos.org/centos/centos:8
 COPY --from=builder /go/src/github.com/code-ready/crc /opt/crc
```
Comment (gbraad) [inline images/openshift-ci/Dockerfile:10]: Nothing?

**Review by gbraad — APPROVED** [review]
The Android client highlighted the whole sentence red based on the Dockerfile statements, like `COPY`, `RUN`, `FROM` hence this looked like removed.

Confirmed this not to be the case. Approving.

**Pull request merged**